src/app/overview/index.js

```diff
@@ -11,17 +11,25 @@ angular
         $scope.overview_md = '(loading)'
 
         projectService.ready(function(project) {
-
+            let project_name = $state.params.project_name
+                ? $state.params.project_name
+                : null;
+            
             // default;
             var selected_overview = project.docs["dbt.__overview__"];
-
             var overviews = _.filter(project.docs, {name: '__overview__'});
-            _.each(overviews, function(overview) {
+            _.each(overviews, function (overview) {
                 if (overview.package_name != 'dbt') {
                     selected_overview = overview;
                 }
             });
-
+            if (project_name !== null) {
+                let project_key = `${project_name}.__${project_name}__`;
```

Comment:
I think we'll want to let top-level projects "override" the project-level overviews of installed packages. If `my_project` installs the `snowplow` package, I would want `my_project.__snowplow__` to take precedence over `snowplow.__snowplow__`, especially if the latter doesn't exist.

Reply:
Thanks a lot @jtcohen6 I'm glad you like the idea, I agree with your comment completely. I added a new commit that I think should handle that scenario. Let me know if you think it needs any other change.